pkg/skaffold/schema/profiles_test.go

```diff
@@ -377,8 +377,10 @@ func TestActivatedProfiles(t *testing.T) {
 				{Name: "activated", Activation: []latest.Activation{{KubeContext: "prod-context"}}},
 				{Name: "not-activated", Activation: []latest.Activation{{KubeContext: "dev-context"}}},
 				{Name: "also-activated", Activation: []latest.Activation{{KubeContext: "!dev-context"}}},
+				{Name: "activated-regexp", Activation: []latest.Activation{{KubeContext: "prod-.*"}}},
+				{Name: "not-activated-regexp", Activation: []latest.Activation{{KubeContext: "dev-.*"}}},
```

Comment:
we should cover the !(regex) and invalid regex cases too.

Reply:
Should I create another profile so I can test those cases more easily?

For instance, in order to test the invalid regex I would need a profile that matches when comparing strings, I guess. Also, is there any character limitation for the profile? I couldn't find any in [create_context.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_context.go)

Regarding the `!regexp` I'm not sure how to test it... Shall we assume that if the regexp is valid we're testing the regexp matching and not the string comparison so I can make a simple test?

Reply:
Feel free to add as many profiles as needed. Also, simple tests are better!